app/scripts/controllers/metamask-notifications/metamask-notifications.ts

```diff
@@ -210,25 +213,25 @@ export type AllowedActions =
   | PushPlatformNotificationsControllerUpdateTriggerPushNotifications;
 
 // Events
-export type MetamaskNotificationsControllerMessengerEvents =
-  ControllerStateChangeEvent<
-    typeof controllerName,
-    MetamaskNotificationsControllerState
-  >;
+export type Events =
+  | MetamaskNotificationsControllerNotificationsListUpdatedEvent
+  | MetamaskNotificationsControllerMarkNotificationsAsRead;
```

Comment:
These events are used for a  feature only available in v12.0.1+, not in v12.0.0.

This is just some cleanup to separate out the controller events vs external events.